core/tee/tee_svc_cryp.c

```diff
@@ -2989,6 +3052,7 @@ TEE_Result tee_svc_asymm_verify(uint32_t state, const TEE_Attribute *params,
 	struct tee_obj *o;
 	size_t hash_size;
 	int salt_len;
+	TEE_Attribute *params = NULL;
```

Comment:
TEE_Attribute looks rather small. Can't we have it on the stack instead of the heap?